semgrep_output_v1.atd

```diff
@@ -1817,6 +1862,7 @@ type function_call
      the RPC pipe.
   *)
   | CallValidate of fpath
+  | CallResolveDependencies of resolve_dependencies_params
```

Comment:
why introduce an intermediate type with a record with just one field?
Why not simpler | CallResolveDependencies of manifest list ?

Reply:
updated!